peripheral.joystick/resources/buttonmaps/xml/linux/Logitech_Inc._WingMan_RumblePad_9b_7a.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" ?>
+<buttonmap>
+    <device name="Logitech Inc. WingMan RumblePad" provider="linux" buttoncount="9" axiscount="7">
+        <configuration>
+            <axis index="1" center="1" range="2" />
```

Comment:
Can you map the two triggers again? This line was probably detected incorrectly.

Reply:
The Logitech Wingman Rumblepad has only one left and one right button.  I assume they are called shoulder buttons as with retropie [here](https://retropie.org.uk/forum/topic/13750/remove-confusion-on-the-terms-trigger-and-shoulder-in-the-documentation/2) and [here](https://retropie.org.uk/docs/Controller-Configuration/).  If kodi uses the opposite convention I'll call them triggers instead of shoulders.  Where's the documentation for shoulder vs trigger in kodi?

That line looks weird.  I'll remap the controller after getting clarification about shoulder vs trigger in kodi so don't merge anything.